suite-native/navigation/src/components/GoBackIcon.tsx

```diff
@@ -6,7 +6,11 @@ export const GoBackIcon = () => {
     const navigation = useNavigation();
 
     const handleGoBack = () => {
-        navigation.goBack();
+        if (navigation.canGoBack()) {
+            navigation.goBack();
+        } else {
+            console.warn('Cannot go back!');
```

Comment:
since we are not handling this the way we show to the user, i would remove the `else`

Reply:
3f1cc053972eb75c24df5b9025eeb6eb72947ac8